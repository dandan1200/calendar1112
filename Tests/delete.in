python calendar.py ADD 10-10-2000 "Birthday Party" "At Home"
python calendar.py ADD 10-10-2000 "Meeting" "At cafe"
python calendar.py ADD 13-10-2000 "New Dinner" "At resteraunt"
python calendar.py GET INTERVAL 10-01-1999 10-10-2001
python calendar.py DEL 10-10-2000 "Meeting"
python calendar.py DEL 13-10-2000 "New Dinner"
python calendar.py GET INTERVAL 10-01-1999 10-10-2001
