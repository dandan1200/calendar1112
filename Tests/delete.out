10-10-2000 : Birthday Party : At Home
10-10-2000 : Meeting : At cafe
13-10-2000 : New Dinner : At resteraunt
10-10-2000 : Birthday Party : At Home
